From c5039552eb8e3b5e92e37f3fc2aa141a8b1ca6f3 Mon Sep 17 00:00:00 2001
From: Basanagouda Nagappa Koppad <Basanagouda.Nagappa.Koppad@intel.com>
Date: Fri, 28 Nov 2025 12:19:02 +0000
Subject: [PATCH] Fix LTS2024 conflict issue

Tracked-on: OAM-xxxx
Signed-off-by: Basanagouda Nagappa Koppad <Basanagouda.Nagappa.Koppad@intel.com>

diff --git a/groups/kernel/AndroidBoard.mk b/groups/kernel/AndroidBoard.mk
index 9d992999..4d094249 100644
--- a/groups/kernel/AndroidBoard.mk
+++ b/groups/kernel/AndroidBoard.mk
@@ -314,6 +314,7 @@ $(foreach v,$(BOARD_DTB_VARIANTS),$(eval $(call board_dtb_per_variant,$(v))))
 {{/build_dtbs}}
 
 ifneq ($(BASE_LTS2024_ANDROID_KERNEL), true)
+ifneq ($(BASE_LINUX_INTEL_LTS2024_KERNEL), true)
 {{#i915_ag_mods_version}}
 
 I915_AG_ADDITIONS_PATH := ../modules/intel-gpu-i915-backports
@@ -332,6 +333,7 @@ $(LOCAL_KERNEL_PATH)/copy_modules: $(I915_AG_MODS_TARGET)
 
 {{/i915_ag_mods_version}}
 endif
+endif
 
 # Add a kernel target, so "make kernel" will build the kernel
 .PHONY: kernel
-- 
2.34.1

