From 352bfa17e4824bb021a9222d0e60c89ae068f497 Mon Sep 17 00:00:00 2001
From: Basanagouda Nagappa Koppad <Basanagouda.Nagappa.Koppad@intel.com>
Date: Fri, 28 Nov 2025 15:44:41 +0000
Subject: [PATCH] Update clang version in kernel config

Fix kernel config errors

Tracked-On: OAM-xxx
Signed-off-by: Basanagouda Nagappa Koppad <Basanagouda.Nagappa.Koppad@intel.com>

diff --git a/groups/kernel/AndroidBoard.mk b/groups/kernel/AndroidBoard.mk
index 4d094249..f8a79017 100644
--- a/groups/kernel/AndroidBoard.mk
+++ b/groups/kernel/AndroidBoard.mk
@@ -190,7 +190,8 @@ $(KERNEL_CONFIG): $(KERNEL_CONFIG_DEPS) | $(CHECK_CONFIG_LOG)
 # BOARD_KERNEL_CONFIG_FILE and BOARD_KERNEL_VERSION can be used to override the values extracted
 # from INSTALLED_KERNEL_TARGET.
 BOARD_KERNEL_CONFIG_FILE = $(KERNEL_CONFIG)
-BOARD_KERNEL_VERSION = $(shell cat $(KERNEL_DEFCONFIG) | sed -nr 's|.*([0-9]+[.][0-9]+[.][0-9]+)(-rc[1-9])? Kernel Configuration.*|\1|p')
+# BOARD_KERNEL_VERSION = $(shell cat $(KERNEL_DEFCONFIG) | sed -nr 's|.*([0-9]+[.][0-9]+[.][0-9]+)(-rc[1-9])? Kernel Configuration.*|\1|p')
+BOARD_KERNEL_VERSION := $(shell grep -o '[0-9]\+\.[0-9]\+\.[0-9]\+' $(KERNEL_DEFCONFIG) | head -1)
 
 $(PRODUCT_OUT)/kernel: $(LOCAL_KERNEL) $(LOCAL_KERNEL_PATH)/copy_modules
 	$(hide) cp $(LOCAL_KERNEL) $@
-- 
2.34.1

